dev/devicelab/bin/tasks/complex_layout_scroll_perf_ios__timeline_summary.dart

```diff
@@ -8,5 +8,5 @@ import 'package:flutter_devicelab/tasks/perf_tests.dart';
 
 Future<void> main() async {
   deviceOperatingSystem = DeviceOperatingSystem.ios;
-  await task(createComplexLayoutScrollPerfTest());
+  await task(createComplexLayoutScrollPerfTest(enableImpeller: true));
```

Comment:
Pass in `enableImpeller` when you want to run both Skia and Impeller.  This parameter needs a rename to make that clearer.